modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/NavigationMenuResourceImpl.java

```diff
@@ -668,6 +797,21 @@ private void _updateNavigationMenuItems(
 		}
 	}
 
+	@Reference
+	private AssetEntryLocalService _assetEntryLocalService;
+
+	@Reference
+	private BlogsEntryLocalService _blogsEntryLocalService;
```

Comment:
Are we using `_blogsEntryLocalService` and `_assetEntryLocalService`?? I'm not seeing it in the code. 😅

Reply:
You are right, they are not used :smile: